searx/static/themes/simple/src/js/main/keyboard.js

```diff
@@ -43,7 +43,7 @@ searxng.ready(function () {
     }
   });
 
-  searxng.on('.result', 'keydown', function (e) {
+  searxng.on('.image-detail-open', 'keydown', function (e) {
     var resultElement = getResultElement(e.target);
     if (isImageResult(resultElement)) {
```

Comment:
If you go with the binding on `.image-detail-open` then this condition isn't required anymore (`.image-detail-open` only exists when an image is opened)